lib/controller.js

```diff
@@ -113,17 +117,25 @@ Tessel.get = function(opts) {
     var seekerOpts = {
       timeout: (opts.timeout || 2) * 1000,
       usb: opts.usb,
-      lan: opts.lan
+      lan: opts.lan,
+      authorized: true
     };
 
+    if (opts.authorized !== undefined) {
+      seekerOpts.authorized = opts.authorized;
+    }
+
     // Create a seeker object and start detecting any Tessels
     var seeker = new discover.TesselSeeker().start(seekerOpts);
+    var noTessels = seekerOpts.authorized ?
+      'No Authorized Tessels Found.' :
+      'No Tessels Found.';
```

Comment:
@Student007 can we try to make this cleaner? It's not good that this is copied and pasted in `Tessel.get` and `Tessel.list`.